File: Java/프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int skewer = 12000*n; //양꼬치 가격
        int service = n/10; //서비스 음료 개수
        int drink = 2000*(k-service); //음료 가격(서비스 음료 제외)
        int answer = skewer+drink;
        return answer;
    }
}

File: Python3/프로그래머스/0/120830. 양꼬치/양꼬치.py
def solution(n, k):
    answer = 0
    skewer = 12000*n #양꼬치 가격
    service = int(n/10) #서비스 음료 개수
    drink = 2000*(k-service) #음료 가격(서비스 음료 제외)
    answer = skewer+drink
    return answer